rwgame/DrawUI.cpp

```diff
@@ -32,26 +32,29 @@ void drawMap(ViewCamera& currentView, PlayerController* player,
              GameWorld* world, GameRenderer* render) {
     MapRenderer::MapInfo map;
 
-    glm::quat camRot = currentView.rotation;
-
-    map.rotation = glm::roll(camRot) - glm::half_pi<float>();
-    map.worldSize = ui_worldSizeMin;
-    map.worldSize = ui_worldSizeMax;
-    if (player) {
-        map.worldCenter = glm::vec2(player->getCharacter()->getPosition());
-    }
+    if (world->state->hudFlash != HudFlash::FlashRadar
```

Comment:
It would be tidier to do this check down in drawHUD, just to avoid changing the indentation here.